src/containers/List/ListFilters/ListFilters.tsx

```diff
@@ -110,10 +116,27 @@ const ListFilters: React.FC<{
               {displayableFilters[filterName].title}
             </Dropdown.Item>
           ))}
-          <Dropdown.Item key="clear-filters" onClick={resetFilters}>
-            <Icon size="small" color="grey" name="delete" />
-            {strings.FILTER_RESET_ALL}
-          </Dropdown.Item>
+          {!!activeFilters.length && (
+            <Dropdown.Item key="clear-filters" onClick={resetFilters}>
+              <Icon size="small" color="grey" name="delete" />
+              {strings.FILTER_RESET_ALL}
+            </Dropdown.Item>
+          )}
+          {query.sortBy && (
+            <Dropdown.Item key="clear-sort" onClick={() => updateQuery({ sortBy: null })}>
+              <Icon size="small" color="grey" name="delete" />
+              {strings.FILTER_RESET_SORT}
+            </Dropdown.Item>
+          )}
+          {(query.page || query.perPage) && (
+            <Dropdown.Item
+              key="clear-pagination"
+              onClick={() => updateQuery({ page: null, perPage: null })}
+            >
+              <Icon size="small" color="grey" name="delete" />
+              {strings.FILTER_RESET_PAGINATION}
+            </Dropdown.Item>
+          )}
```

Comment:
This is the "Reset Filters", "Reset Sort" and "Reset pagination" controls. I've made them all only appear if there's actually something to reset. Does that seem okay, or do you reckon it's better to have them in the menu all the time regardless?